exercises/proverb/Sources/ProverbExample.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+struct Proverb {
+
+    func recite(_ pieces: [String]) -> String {
```

Comment:
This should probably be an extension on `Array where Element:String`

Reply:
I don't feel this is necessary...

Reply:
But it can be done if needed...

Reply:
Ok I did it.